Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -10,8 +13,18 @@ public class TelepathyTransport : Transport
         [Tooltip("Nagle Algorithm can be disabled by enabling NoDelay")]
         public bool NoDelay = true;
 
+        [EditorBrowsable(EditorBrowsableState.Never), System.Obsolete("Use MaxMessageSizeFromClient or MaxMessageSizeFromServer instead.")]
```

Comment:
please import System and use [Obsolete] instead of [System.Obsolete] for consistency with all the other obsoletes.